src/sentry/templates/sentry/emails/activity/generic.html

```diff
@@ -4,14 +4,28 @@
 {% load sentry_assets %}
 
 {% block header %}
-  <a href="{{ link }}" class="btn">View on Sentry</a>
+  {% block action %}
+    <a href="{{ link }}" class="btn">View on Sentry</a>
+  {% endblock %}
   {{ block.super }}
 {% endblock %}
 
 {% block main %}
-  <h2>{{ activity_name }}</h2>
 
-  <p>{{ html_description }}</p>
+  {% block activity %}
+
+    <h2>{{ activity_name }}</h2>
+
+    <p>{{ html_description }}</p>
+
+  {% endblock %}
 
   {% include "sentry/emails/group_header.html" %}
+
+  {% if reason %}
```

Comment:
This conditional shouldn't be necessary, but added it just to be safe and ensure this didn't result in a sentence fragment.,